src/main/java/org/opentripplanner/netex/mapping/StopMapper.java

```diff
@@ -44,23 +56,92 @@ Stop mapQuayToStop(
       return null;
     }
 
+    var wheelchairBoarding = wheelChairBoardingFromQuay(quay, parentStation);
+
     Stop stop = new Stop(
-        idFactory.createId(quay.getId()),
-        parentStation.getName(),
-        quay.getPublicCode(),
-        quay.getDescription() != null ? quay.getDescription().getValue() : null,
-        WgsCoordinateMapper.mapToDomain(quay.getCentroid()),
-        null,
-        null,
-        null,
-        fareZones,
-        null,
-        null,
-        transitMode.first,
-        transitMode.second
+            idFactory.createId(quay.getId()),
+            parentStation.getName(),
+            quay.getPublicCode(),
+            quay.getDescription() != null ? quay.getDescription().getValue() : null,
+            WgsCoordinateMapper.mapToDomain(quay.getCentroid()),
+            wheelchairBoarding,
+            null,
+            null,
+            fareZones,
+            null,
+            null,
+            transitMode.first,
+            transitMode.second
     );
+
     stop.setParentStation(parentStation);
 
     return stop;
   }
+
+  /**
+   * Get WheelChairBoarding from Quay and parent Station.
+   *
+   * @param quay          NeTEx quay could contain information about accessability
+   * @param parentStation Used to get a default WheelChairBoarding if not found from Quay.
+   * @return not null value with default NO_INFORMATION if nothing defined in quay or
+   * parentStation.
+   */
+  private WheelChairBoarding wheelChairBoardingFromQuay(Quay quay, Station parentStation) {
+
+    var defaultWheelChairBoarding = WheelChairBoarding.NO_INFORMATION;
+    var stopPlaceId = parentStation.getId();
+    var stopPlace = stopPlaceIndex.lookupLastVersionById(stopPlaceId.getId());
+
+    if (stopPlace != null) {
+      defaultWheelChairBoarding = wheelChairBoarding(
+              stopPlace.getAccessibilityAssessment(),
+              WheelChairBoarding.NO_INFORMATION
+      );
+    }
+
+    return wheelChairBoarding(quay.getAccessibilityAssessment(), defaultWheelChairBoarding);
+  }
+
+  /**
```

Comment:
The code below should be in it own mapper, the mapping methods can probably be static.

Reply:
Done